packages/flutter_arc_text/lib/src/arc_text.dart

```diff
@@ -95,27 +95,39 @@ class ArcText extends StatelessWidget {
   final PainterDelegate painterDelegate;
 
   @override
-  Widget build(BuildContext context) => CustomPaint(
-        painter: _Painter(
-          radius: radius,
-          text: text,
-          textStyle: textStyle,
-          alignment: startAngleAlignment,
-          initialAngle: startAngle,
-          direction: direction,
-          placement: placement,
-          stretchAngle: stretchAngle,
-          interLetterAngle: interLetterAngle,
-          painterDelegate: painterDelegate,
-        ),
-      );
+  Widget build(BuildContext context) {
+    final DefaultTextStyle defaultTextStyle = DefaultTextStyle.of(context);
+    TextStyle? effectiveTextStyle = textStyle;
+    if (textStyle == null || textStyle!.inherit) {
+      effectiveTextStyle = defaultTextStyle.style.merge(textStyle);
+    }
+    if (MediaQuery.boldTextOf(context)) {
```

Comment:
I don't think it should be the responsibility of this widget to take into account media query data. It should just take whatever `DefaultTextStyle` defines.

Reply:
This is literally what Text widget does.  So I was keeping the behaviour the same.

Reply:
Hm, yeah, looks a bit weird to me, but let's keep it consistent with `Text` widget.